app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <h4><%= _('Puppet CA Details') %></h4>
+    </div>
+    <%= render :partial => 'smart_proxies/plugins/plugin_version', :locals => {:feature => feature, :proxy => @smart_proxy } %>
```

Comment:
I think the certificates should be a tab added when the PuppetCA feature is present